ilastik/applets/pixelClassification/pixelClassificationGui.py

```diff
@@ -784,28 +800,53 @@ def hasFeatures(self) -> bool:
     def isLiveUpdateEnabled(self):
         return self.labelingDrawerUi.liveUpdateButton.isChecked()
 
-    def setLiveUpdateEnabled(self, checked: bool):
-        if checked:
+    def _update_live_update_button(self, enable: bool):
```

Comment:
Could inline argument since `enable == not self.topLevelOperatorView.FreezePredictions.value`.

Reply:
Not sure whether we could do any better, but this method's name is awkward...

Reply:
> Not sure whether we could do any better, but this method's name is awkward...

oh wow, i didn't even realize how bad the name was; thx ;)